frontend/src/components/doctors-filtration-panel/doctors-filtration-panel.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { useForm } from 'react-hook-form';
+import styles from './doctors-filtration.module.scss';
+import { TextInput, Checkbox } from 'components/common';
+import { DoctorType, ClinicType, InputType, InputColor, DoctorFiltration, Icons } from 'common/enums';
+import { IDoctorFiltrationPayload } from 'common/interfaces';
+import { DEFAULT_VALUES } from './common/constants/payload-default-values'
+import Details from 'components/common/details/details';
+
+const doctorSpecialties = Object.keys(DoctorType);
+const clinicTypes = Object.keys(ClinicType);
+
+const DoctorsFiltration: React.FC = () => {
+  const { handleSubmit, control, errors } = useForm<IDoctorFiltrationPayload>({
+    defaultValues: DEFAULT_VALUES,
+    mode: "onChange",
+  });
+
+  const handleSubmitForm = (formData: IDoctorFiltrationPayload) => {
+    /* eslint-disable-next-line no-console */
+    console.log(formData);
+
+  };
+
+  return (
+    <form onSubmit={handleSubmit(handleSubmitForm)}>
+      <div className={styles.panel}>
+        <div className={styles.filters}>
+          <div className={styles.filterHeader}>Filter by</div>
+          <div className={styles.commonFilter}>
+            <TextInput
+              name={DoctorFiltration.DOCTOR_NAME}
+              label="Search by doctor's name"
+              hasHiddenLabel={false}
```

Comment:
We non't have label in the design
```suggestion
              label="Search by doctor's name"
              hasHiddenLabel
```